src/hooks/useCurrentVideoRoom.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  currentVideoRoomSelector,
+  isCurrentVideoRoomRequestedSelector,
+} from "utils/selectors";
+
+import { useFirestoreConnect } from "./useFirestoreConnect";
+import { useSelector } from "./useSelector";
+
+export const useCurrentVideoRoom = (roomId: string) => {
```

Comment:
> Is this guaranteed to always exist where this hook is called? Will we need to make it optional and use our 'only connect if present' pattern here?

From UX perspective, it's impossible for this to be `undefined` however, if someone decides to copy/paste a link with videoRoomId in the url. This might be undefined and render null